gdspx.go

```diff
@@ -179,18 +183,41 @@ func initSpritePhysicInfo(sprite *SpriteImpl, proxy *engine.ProxySprite) {
 		proxy.SetTriggerEnabled(true)
 		proxy.SetTriggerRect(sprite.triggerCenter.ToVec2(), sprite.triggerSize.ToVec2())
 	case physicColliderAuto:
-		w, h := sprite.getCostumeSize()
-		w, h = w*sprite.scale, h*sprite.scale
+		sprite.triggerCenter, sprite.triggerSize = getCostumeBoundByAlpha(sprite, sprite.scale)
 		proxy.SetTriggerEnabled(true)
-		sprite.triggerSize = *math32.NewVector2(w, h)
-		sprite.triggerCenter = *math32.NewVector2(0, 0)
 		proxy.SetTriggerRect(sprite.triggerCenter.ToVec2(), sprite.triggerSize.ToVec2())
 	case physicColliderNone:
 		proxy.SetTriggerEnabled(false)
 	}
 
 }
 
+func getCostumeBoundByAlpha(p *SpriteImpl, pscale float64) (math32.Vector2, math32.Vector2) {
+	cs := p.costumes[p.costumeIndex_]
+	// GetBoundFromAlpha is very slow, so we should cache the result
+	if cache, ok := cachedBounds[cs.path]; ok {
```

Comment:
> The same `path` can be used in different costumes with different `bitmapResolution` values. It seems more reliable to cache the result of `gdspx.ResMgr.GetBoundFromAlpha` which is determined solely by `path`, rather than the result of `getCostumeBoundByAlpha`.

Fixed by this commit : https://github.com/goplus/spx/pull/448/commits/6be73b3318cc6aa908d8d275aa35bb538e68d5ba